### PR TITLE
Remove unused OIDs (LG-12952)

### PR DESCRIPTION
**Pull request by mdiarra3**
## 🎫 Ticket
[LG-12952](https://cm-jira.usa.gov/browse/LG-12952)

## 🛠 Summary of changes

Remove unused OIDs

### CODE REVIEW

**Review by kevinsmaster5 — APPROVED** [review]
Looks good 👍

File: config/application.yml.default [hunk 1]
```diff
@@ -41,19 +41,9 @@ trusted_ca_root_identifiers: "F4:27:5C:A9:C3:7C:47:F4:FA:A6:A7:B0:59:97:AA:DD:35
...
-    "2.16.840.1.101.2.1.11.19",
-    "2.16.840.1.101.2.1.11.42",
-    "2.16.840.1.101.2.1.11.43",
-    "2.16.840.1.101.2.1.11.44",
     "2.16.840.1.101.3.2.1.3.7",
     "2.16.840.1.101.3.2.1.3.13",
-    "2.16.840.1.101.3.2.1.12.2",
-    "2.16.840.1.101.3.2.1.12.3",
-    "2.16.840.1.101.3.2.1.12.5",
     "2.16.840.1.101.3.2.1.12.6",
```
Comment (mitchellhenke) [inline config/application.yml.default:46]:
```suggestion
    "2.16.840.1.101.3.2.1.12.6"
```

**Review by mitchellhenke — APPROVED** [review]

File: spec/controllers/identify_controller_spec.rb [hunk 2]
```diff
@@ -370,9 +370,9 @@
               valid: false,
               error: 'self-signed cert',
               openssl_valid: false,
-              openssl_errors: 'error 18 at 0 depth lookup: self signed certificate, error 26 at 0 depth lookup: unsupported certificate purpose',
```
Comment (mitchellhenke) [inline spec/controllers/identify_controller_spec.rb:373]: This test is for self-signed certificates, so I'm surprised that it needs to have the self-signed part of the error message removed.

Comment (mdiarra3) [reply spec/controllers/identify_controller_spec.rb:373]: yea me too. Gonna check on this more

**Pull request merged**